src/github.com/stellar/gateway/bridge/config/config.go

```diff
@@ -68,6 +68,49 @@ func (c *Config) Validate() (err error) {
 		return
 	}
 
+	for _, asset := range c.Assets {
+		if asset.Issuer == "" {
+			if asset.Code != "XLM" {
+				err = errors.New("Issuer param is required for "+asset.Code)
+				return
+			}
+		}
+
+		if asset.Issuer != "" {
+			_, err = keypair.Parse(asset.Issuer)
+			if err != nil {
+				err = errors.New("Issuing account is invalid for "+asset.Code)
+				return
+			}
+		}
+
+		if asset.Code == "" {
+			err = errors.New("Asset code is required")
+			return
+		}
+
+		if len(asset.Code) > 12 {
```

Comment:
Do you know of a better way to check the type? Is there a static variable I can use instead?